packages/yoga/src/Dialog/web/Dialog.jsx

```diff
@@ -83,23 +83,26 @@ const Dialog = ({ isOpen, children, onClose }) => {
     return () => document.removeEventListener('keydown', keyPress);
   }, [keyPress]);
 
-  return isOpen ? (
-    createPortal(
-      <Overlay onClick={closeDialog} onClose={onClose} ref={dialogRef}>
-        <StyledDialog onClose={onClose}>
-          {onClose && (
-            <Box d="flex" justifyContent="flex-end" w="100%">
-              <Button.Icon icon={Close} inverted onClick={onClose} />
-            </Box>
-          )}
-          {children}
-        </StyledDialog>
-      </Overlay>,
-      dialogElement,
-    )
-  ) : (
-    <> </>
-  );
+  return isOpen
+    ? createPortal(
+        <Overlay
+          onClick={closeDialog}
+          onClose={onClose}
+          ref={dialogRef}
+          {...props}
+        >
+          <StyledDialog onClose={onClose}>
+            {onClose && (
+              <Box d="flex" justifyContent="flex-end" w="100%">
+                <Button.Icon icon={Close} inverted onClick={onClose} />
+              </Box>
+            )}
+            {children}
+          </StyledDialog>
+        </Overlay>,
+        dialogElement,
+      )
+    : null;
```

Comment:
I believe it can generate a problem with null, what do you think  @coelhucas ?

Reply:
I change this to:

<img width="111" alt="Screen Shot 2021-11-16 at 09 03 43" src="https://user-images.githubusercontent.com/27781419/141982156-ebe9e98a-af20-4e58-b2b0-a2306cc1ce39.png">